skops/hub_utils/tests/test_hf_hub.py

```diff
@@ -39,13 +94,13 @@ def test_validate_folder():
     ):
         _validate_folder(path=dir_path)
 
-    example_file = _get_cwd() / "sample_repo/config.json"
+    example_file = _get_cwd() / "sample_repo" / "config.json"
```

Comment:
instead of `_get_cwd() / "sample_repo"` we could use a fixture returning a temp folder.

Reply:
good catch, will change it